vcpkg.json

```diff
@@ -9,7 +9,9 @@
       "name": "curl",
       "features": [
         "brotli",
-        "http2"
+        "http2",
+        "ssl",
+        "websockets"
```

Comment:
idk if distros ship libcurl with websockets enabled, but they hopefully will after they try to package us 😅